src/components/ChatLoader/styles.scss

```diff
@@ -1,15 +1,25 @@
 // Chat Loader.scss
 
 .sg.typing-loader {
-    background-color: #f1f1f1;
-    
-    color: var(--gray-3);
-    }
-
-  @keyframes blink { 50% { fill: transparent }}
-  .dot { 
-    animation: 1s blink infinite;
-    fill: var(--gray-3);
+  background-color: #f1f1f1;
+  color: var(--gray-3);
+}
+
+@keyframes blink {
+  50% {
+    fill: transparent;
   }
-  .dot:nth-child(2) { animation-delay: 250ms }
-  .dot:nth-child(3) { animation-delay: 500ms }
\ No newline at end of file
+}
+
+.dot {
```

Comment:
We can't use `.dot` directly outside without the parent. 

It might affect some other CSS somewhere else within our app/ 3rd party library.